open_academy/models/course.py

```diff
@@ -0,0 +1,23 @@
+from odoo import models, fields
+
+
+class Course(models.Model):
+    _name = "openacademy.course"
+    _description = "Courses"
+
+    title = fields.Char(string="Courses", required=True)
+    description = fields.Text()
+
+    responsible_user = fields.Many2one(comodel_name="res.users", ondelete="set null")
```

Comment:
You don't need to add comodel_name

```suggestion
    responsible_user = fields.Many2one("res.users", ondelete="set null")
```